scvi/model/base/_base_model.py

```diff
@@ -119,6 +140,16 @@ def _make_data_loader(
         data_loader_kwargs
             Kwargs to the class-specific data loader class
         """
+        if adata is None:
+            adata_manager = self.adata_manager
+        else:
+            adata_manager = self.get_anndata_manager(adata)
+            if adata_manager is None:
```

Comment:
should we proceed here with transfer? This function is typically run after validation. See get latent rep for example...

Reply:
perhaps we raise an error to go run validate?

Reply:
i see, that sounds good